pages/signInPage.tsx

```diff
@@ -0,0 +1,121 @@
+              </div>
+              <div className="justify-center mt-2">
+                <p className="text-white text-lg font-sans subpixel-antialiase tracking-wide font-bold">
+                  Welcome to the Membership Portal. Here you can find&nbsp;&nbsp;&nbsp; <br />
+                  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;information about our exclusive events, open <br />
+                  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;applications, and attendance information
+                </p>
+              </div>
+            </div>
+
+            <div className="absolute inset-x-0 bottom-10 flex flex-row justify-center space-x-8">
+              <div>
+                <svg width="40" height="40" viewBox="0 0 41 40" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <a href="https://discord.com/invite/7fZQZyP">
+                    <path d="M33.7693 8.61568C30.6137 6.05318 25.6216 5.6188 25.408 5.60318C25.0766 5.57505 24.7607 5.76255 24.6245 6.07193C24.6121 6.09068 24.5037 6.3438 24.3829 6.73755C26.4702 7.0938 29.0343 7.80943 31.3538 9.26255C31.7254 9.4938 31.84 9.98755 31.6108 10.3626C31.4591 10.6094 31.2021 10.7438 30.9357 10.7438C30.7933 10.7438 30.6477 10.7032 30.5177 10.6219C26.529 8.12505 21.5494 8.00005 20.5894 8.00005C19.6294 8.00005 14.6466 8.12505 10.6611 10.6219C10.2894 10.8563 9.80015 10.7407 9.57099 10.3657C9.33873 9.98755 9.45331 9.49693 9.82492 9.26255C12.1444 7.81255 14.7086 7.0938 16.7958 6.74068C16.675 6.3438 16.5666 6.0938 16.5573 6.07193C16.418 5.76255 16.1052 5.5688 15.7708 5.60318C15.5571 5.6188 10.5651 6.05318 7.36607 8.65005C5.6969 10.2094 2.35547 19.3219 2.35547 27.2001C2.35547 27.3407 2.39263 27.4751 2.46076 27.5969C4.76477 31.6844 11.0543 32.7532 12.4882 32.8001C12.4944 32.8001 12.5036 32.8001 12.5129 32.8001C12.7669 32.8001 13.0053 32.6782 13.154 32.4719L14.6033 30.4594C10.692 29.4407 8.6946 27.7094 8.58002 27.6063C8.25176 27.3157 8.22079 26.8094 8.51189 26.4782C8.79989 26.1469 9.30157 26.1157 9.62983 26.4063C9.67628 26.4501 13.3553 29.6001 20.5894 29.6001C27.8358 29.6001 31.5148 26.4376 31.552 26.4063C31.8803 26.1188 32.3788 26.1469 32.6699 26.4813C32.9579 26.8126 32.927 27.3157 32.5987 27.6063C32.4841 27.7094 30.4867 29.4407 26.5755 30.4594L28.0248 32.4719C28.1734 32.6782 28.4118 32.8001 28.6658 32.8001C28.6751 32.8001 28.6844 32.8001 28.6906 32.8001C30.1244 32.7532 36.414 31.6844 38.718 27.5969C38.7861 27.4751 38.8233 27.3407 38.8233 27.2001C38.8233 19.3219 35.4818 10.2094 33.7693 8.61568ZM15.4363 24.0001C13.9034 24.0001 12.6616 22.5688 12.6616 20.8001C12.6616 19.0313 13.9034 17.6001 15.4363 17.6001C16.9692 17.6001 18.211 19.0313 18.211 20.8001C18.211 22.5688 16.9692 24.0001 15.4363 24.0001ZM25.7424 24.0001C24.2095 24.0001 22.9677 22.5688 22.9677 20.8001C22.9677 19.0313 24.2095 17.6001 25.7424 17.6001C27.2753 17.6001 28.5171 19.0313 28.5171 20.8001C28.5171 22.5688 27.2753 24.0001 25.7424 24.0001Z" fill="white" />
```

Comment:
Suggestion, can we have it so that on hover the icon will color up with the actual logo color?

Reply:
Is there a suggested way to do this? Couldn't figure this out.